### PR TITLE
PE loader now adds an entry for the main executable to LDR_DATA

### PR DESCRIPTION
**Pull request by jhumble**
I added a few lines of code to insert a LDR_MODULE entry for the main exe being run to match the structure on Windows. Prior to this fix, I was getting crashes on some packers/malware that were crawling the ldr data as a method of finding PE_IMAGE_BASE.

Also made a small fix to handle null parameters to SetClipboardData and removed a couple lines of debug code that made their way into windows_trace.py

### CODE REVIEW

File: qiling/os/windows/dlls/user32.py [hunk 1]
```diff
@@ -113,7 +113,11 @@ def hook_CloseClipboard(ql, address, params):
     "hMem": STRING
 })
 def hook_SetClipboardData(ql, address, params):
-    return ql.clipboard.set_data(params['uFormat'], bytes(params['hMem'], 'ascii'))
+    try:
+        data = bytes(params['hMem'], 'ascii')
```
Comment (cq674350529) [inline qiling/os/windows/dlls/user32.py]: The type of `data` is inconsistent. Maybe using `data=b""` instead?

File: qiling/os/windows/dlls/user32.py [hunk 2]
```diff
@@ -113,7 +113,11 @@ def hook_CloseClipboard(ql, address, params):
...
+    except:
+        data = ""
```
Comment (cq674350529) [inline qiling/os/windows/dlls/user32.py]: The type of `data` is inconsistent. Maybe using `data=b""` instead?

**Comment by xwings** [comment]
@jhumble if you agree, you can update accodringly ?

File: qiling/os/windows/dlls/user32.py (hunk 2 above)
Comment (jhumble) [reply qiling/os/windows/dlls/user32.py]: You're right. I fixed it. Thank you.

**Comment by Dliv3** [comment]
Thanks! Merged

**Pull request merged**